avatars/processors/group_modalities.py

```diff
@@ -95,5 +95,5 @@ def preprocess(self, df: pd.DataFrame) -> pd.DataFrame:
 
         return df
 
-    def postprocess(self, df: pd.DataFrame) -> pd.DataFrame:
-        return df
+    def postprocess(self, source: pd.DataFrame, dest: pd.DataFrame) -> pd.DataFrame:
+        return dest
```

Comment:
what about a `general processor test` whose porpose is to verify : 
- the processor has a preprocess and a post processor 
- fit the arguements
- other stuff